lib/actionmailer-text/html_to_plain_text.rb

```diff
@@ -31,6 +31,9 @@ def self.convert_to_text(html, line_length = 65, _from_charset = 'UTF-8')
         txt.gsub!(/<img.+?alt=\"([^\"]*)\"[^>]*\/>/i, '\1')
         txt.gsub!(/<img.+?alt='([^\']*)\'[^>]*\/>/i, '\1')
 
+        # remove empty links
```

Comment:
I wonder if this can be combined with what's below and simply check that the text is not empty?